pysat/instruments/pysat_testmodel.py

```diff
@@ -118,9 +113,16 @@ def load(fnames, tag=None, inst_id=None, num_samples=None):
                                      '(zenith angle of sun) of the given ',
                                      'locaiton. Overhead noon, +/- 90 is 6, ',
                                      '18 SLT .'])}
+    meta['mlt'] = {'units': 'hours', 'long_name': 'Solar Local Time',
+                   'desc': 'Magentic Local Time', 'value_min': 0.0,
+                   'value_max': 24.0}
     meta['longitude'] = {'units': 'degrees', 'long_name': 'Longitude'}
```

Comment:
min/max/description (Geographic Longitude). Since meta is already being updated and we are approaching a big release I think the information should be as complete as possible.

Reply:
I think this could be pushed to 3.1.0, when all of this goes into a single routine.